src/cmd/initialize.go

```diff
@@ -114,7 +115,7 @@ func downloadInitPackage(downloadCacheTarget string) error {
 	}
 
 	var confirmDownload bool
-	url := packager.GetInitPackageRemote("")
+	url := oci.GetInitPackageURL(config.GetArch(), config.CLIVersion)
 
 	// Give the user the choice to download the init-package and note that this does require an internet connection
 	message.Question(fmt.Sprintf(lang.CmdInitDownloadAsk, url))
```

Comment:
One tiny little thing here, when we print this to the user can we show them the `oci://` url as they would use it on package deploy?

i.e.

```
It seems the init package could not be found locally, but can be pulled from
oci://docker.io/racer159/init:v0.29.0-17-g046e436a-amd64

 NOTE  Note: This will require an internet connection.
? Do you want to pull this init package? Yes
```
![image](https://github.com/defenseunicorns/zarf/assets/3977569/fdecbca4-a6ee-4e5b-a183-7a15ce185ed3)

(Not this adds `oci://` and changes `download` to `pull` in this context

Reply:
lmk if this works